component/otelcol/receiver/vcenter/vcenter_test.go

```diff
@@ -36,7 +174,53 @@ func TestArguments_UnmarshalRiver(t *testing.T) {
 	require.Equal(t, time.Second, otelArgs.ScraperControllerSettings.InitialDelay)
 	require.Equal(t, 0*time.Second, otelArgs.ScraperControllerSettings.Timeout)
 
-	require.Equal(t, true, otelArgs.Metrics.VcenterClusterCPUEffective.Enabled)
-	require.Equal(t, false, otelArgs.Metrics.VcenterVMMemoryUtilization.Enabled)
+	// Verify ResourceAttributesConfig fields
 	require.Equal(t, true, otelArgs.ResourceAttributes.VcenterClusterName.Enabled)
+	require.Equal(t, true, otelArgs.ResourceAttributes.VcenterDatastoreName.Enabled)
+	require.Equal(t, true, otelArgs.ResourceAttributes.VcenterHostName.Enabled)
+	require.Equal(t, false, otelArgs.ResourceAttributes.VcenterResourcePoolInventoryPath.Enabled)
+	require.Equal(t, true, otelArgs.ResourceAttributes.VcenterResourcePoolName.Enabled)
+	require.Equal(t, true, otelArgs.ResourceAttributes.VcenterVMName.Enabled)
+	require.Equal(t, true, otelArgs.ResourceAttributes.VcenterVMID.Enabled)
+
+	// Verify MetricsConfig fields
+	require.Equal(t, false, otelArgs.Metrics.VcenterClusterCPUEffective.Enabled)
+	require.Equal(t, true, otelArgs.Metrics.VcenterClusterCPULimit.Enabled)
```

Comment:
```suggestion
	require.False(t, otelArgs.Metrics.VcenterClusterCPUEffective.Enabled)
	require.True(t, otelArgs.Metrics.VcenterClusterCPULimit.Enabled)
```

I think this is clearer, but I'm happy if you leave it as it is.

Reply:
done